docs/proposals/accepted/observatorium.md

```diff
@@ -0,0 +1,222 @@
+
+## Goals
+
+**Target Audience**: Engineers who want to provide observability services for monitoring, debugging and SLOs assurance for themselves, their teams or even SaaS customers.
+
+An Observability System that has the following characteristics:
+
+* Open Source first
+  * Apache v2 license
+  * Free to use
+  * All (in-scope) contributions welcome.
+* Ability to scrape/ingest (write), store, query (read), and retrieve metrics, logs and traces from multiple clusters and tenants. APIs for now:
+  * Metrics
+    * Write:
+      * OpenMetrics (pull model)
+      * Prometheus Remote Write API (push model)
+    * Read:
+      * Compatible with the Prometheus query APIs (e.g PromQL) making it possible to use any dashboard solution (e.g. Grafana).
+  * Logs: Kubernetes and Pod logs (Loki API)
+* Support recording rules and alerting based on data from multiple clusters and tenants.
+  * Allow users to configure alerts and recording rules with immediate effect (minutes)
+* Unified installation and deployment model for Kubernetes and OpenShift using Operator, OLM, non Operator, helm and Openshift Templates. 
+* Support multi-tenancy for all APIs allowing to choose between soft and hard tenancy.
+  * Multiple tenants can access the same metrics (cross tenancy)
+* Support centralized and in-cluster deployments (or mix) for all APIs. In particular, this means the following topologies:
+  * Single Cluster (everything runs inside the cluster).
+  * Mix of in cluster monitoring/alerting and subset of/all metrics pushed to a central location.
+  * The purely centralized push model
+* Horizontally Scalable: supporting smaller as well as larger scale.
+  * It should be possible to deploy the solution in small environments such as single-node clusters.
+  * Scaling up is a matter of adding more instances rather than throwing more resources at the same instance.
+* Relatively pp, easy to operate, and ubiquitous storage even for long retention:
+  * Object storage is the main storage solution for longer retention.
+  * Block storage is only for short term data or local buffers.
+* Ability to define data retention policies.
+* Control over the granularity of the data access.
+  * Allowing a tenant to read a subset of data.
+  * Allowing a tenant to send a subset of the data off the cluster.
+* Secure by default
+  * All external-facing APIs support TLS for encryption
+  * All external-facing APIs support OpenID for authentication.
+  * All external-facing APIs support [OPA](https://www.openpolicyagent.org/) for authorization.
+* High availability and solid failover handling.
+  * For example, for the write path, the client side can survive the unavailability of a central cluster for e.g 30m without data loss.
+* Built-in mechanisms to support production setups out of the box.
+  * Rate-limiting.
+  * Retries.
+  * Sane defaults for metrics ingestion (remote write configuration, sharding, queues).
+* Ability to maintain alert routing (e.g Alertmanager).
```

Comment:
Is Alertmanager also multi-tenant in observatorium? Is it configurable for users?

Reply:
Not at present - though it is one of the obvious big features that users will ask for.

Reply:
Moving t to soft goals